docs/docs/reference/changed-features/structural-types.md

```diff
@@ -123,6 +123,43 @@ New instances of `Selectable` can be defined to support means of
 access other than Java reflection, which would enable usages such as
 the database access example given at the beginning of this document.
 
+## Local Selectable Instances
+
+Local and anonymous classes that extend `Selectable` get more refined types
+than other classes. Here is an example:
+```scala
+class Vehicle extends reflect.Selectable {
+  val wheels: Int
+}
+val i3 = new Vehicle { // i3: Vehicle { val range: Int }
+  val wheels = 4
+  val range = 240
+}
+i3.range
+```
+The type of `i3` in this example is `Vehicle { val range: Int }`. Hence,
+`i3.range` is well-formed. Since the base class `Vehicle` does not define a `range` field or method, we need structural dispatch to access the `range` field of the anonymous class that initializes `id3`. Structural dispatch
+is implemented by the base trait `reflect.Selectable` of `Vehicle`, which
+defines the necessary `selectDynamic` member.
+
+`Vehicle` could also extend some other subclass of `scala.Selectable` that implements `selectDynamic` and `applyDynamic` differently. But if it does not extend a `Selectable` at all, the code would no longer typecheck:
```

Comment:
here you write `scala.Selectable` but above you write `reflect.Selectable` (and also in the tests). I assume this is a typo, but perhaps `scala.Selectable` is also a thing.

Reply:
Yes, scala.reflect.Selectable is the java-reflection-based implementation of the scala.Selectable mechanism https://github.com/lampepfl/dotty/blob/master/library/src/scala/Selectable.scala  https://github.com/lampepfl/dotty/blob/master/library/src/scala/reflect/Selectable.scala

Reply:
Got it. Thanks for the clarification. (I never use these due to performance anxiety associated with reflection, so I had forgotten about that).